api/src/main/java/io/opencensus/internal/OpenCensusThreadFactory.java

```diff
@@ -31,22 +33,27 @@
   private static final ThreadFactory threadFactory = MoreExecutors.platformThreadFactory();
   private final AtomicInteger threadIdGen = new AtomicInteger();
   private final String threadPrefix;
+  private final Boolean startAsDaemon;
 
   /**
-   * Constructs a new {@code DaemonThreadFactory}.
+   * Constructs a new {@code OpenCensusThreadFactory}.
    *
    * @param threadPrefix used to prefix all thread names. (E.g. "CensusDisruptor").
+   * @param startAsDaemon whether to start new threads as daemon.
    */
-  public DaemonThreadFactory(String threadPrefix) {
+  public OpenCensusThreadFactory(String threadPrefix, Boolean startAsDaemon) {
```

Comment:
I am trying to make it a general configurable ThreadFactory so that it can create non-daemon threads.

But I think this is no longer valid now since we are trying to constrain the change within exporter side. I will leave the file as it was and create a new ThreadFactory in stats exporter.